content/pages/about-en.rst

```diff
@@ -0,0 +1,23 @@
+À propos
+########
+
+:slug: a-propos
+:lang: en
+:url: about
+
+We are the "Cloud-services" french team at
+`Mozilla <https://www.mozilla.com/>`_:
+`n1k0 <https://nicolas.perriault.net/>`_,
+`Mathieu <http://mathieu-leplatre.info>`_, `Tarek <http://ziade.org/>`_,
+`Rémy <https://twitter.com/Natim>`_ and `Alexis <http://notmyidea.org>`_.
+
+After some time spend accumulating experience without writting about
+it, we decided to start this blog, where we write in French and in
+English, about our explorations and reflections.
+
+We encourage you, the reader, to interact criticise and give feedback
+about what we write in order for us to improve the outcome of what we
+are trying to build with you.
```

Comment:
This sentence seems a bit heavy. What about: "Don't hesitate to interact and criticize: we want to build this with the help of the broader community (you!)".